packages/cdk-routing-split/src/delete/handler.ts

```diff
@@ -0,0 +1,25 @@
+import { APIGatewayEvent, Context } from 'aws-lambda';
+import { createLogger, Logger, serializeError } from '@daysmart/aws-lambda-logger';
+import { HttpError } from '../http-error';
+import { action } from './action';
+
+export const handler = async (event: APIGatewayEvent, context: Context): Promise<any> => {
+    let logger!: Logger;
+
+    try {
+        logger = createLogger(process.env.DEBUG === 'true', context.awsRequestId);
```

Comment:
How we are setting environment variables, in this case DEBUG. Seams like files infrastructure/config.sandbox.yml and infrastructure/config.sandbox2.yml are missing?

Reply:
this is going to be handled by Chris once this is merged. I removed them because it didn't seem like i was using it in anyway.